### DISCUSSION TITLE
Performing CEF Conformance test

### DISCUSSION THREAD

**Discussion by Kim-Billit** [OxalisCommunity/Oxalis-AS4#153] [Q&A]
I am trying to perform the CEF Conformance tests so they can confirm our solution supports en16931 UBL and CII messages.
To be clear this is a different test then the CEF AS4 Connectivity test, but we are also using AS4 to perform the test.

After being in contact with CEF Support they told me I have to remove the SBDH, and be able to set the service action and the sender and receiver role.

In the soap message this is defined as followed:

            <ns:PartyInfo>
               <ns:From>
                  <ns:PartyId type="**urn:oasis:names:tc:ebcore:partyid-type:unregistered**">einvoicingct_31_gw</ns:PartyId>
                  **<ns:Role>http://docs.oasis-open.org/ebxml-msg/ebms/v3.0/ns/core/200704/initiator</ns:Role>**
               </ns:From>
               <ns:To>
                  <ns:PartyId type="**urn:oasis:names:tc:ebcore:partyid-type:unregistered**">domibus-gitb</ns:PartyId>
                  <ns:Role>**http://docs.oasis-open.org/ebxml-msg/ebms/v3.0/ns/core/200704/responder**</ns:Role>
               </ns:To>
            </ns:PartyInfo>
            <ns:CollaborationInfo>
               **<ns:Service type="tc1">bdx:noprocess</ns:Service>**
               **<ns:Action>TC1Leg1</ns:Action>**
            </ns:CollaborationInfo>
            <ns:MessageProperties>
               **<ns:Property name="originalSender">urn:oasis:names:tc:ebcore:partyid-type:unregistered:C1</ns:Property>**
               **<ns:Property name="finalRecipient">urn:oasis:names:tc:ebcore:partyid-type:unregistered:C4</ns:Property>**
            </ns:MessageProperties>

How can I configure this using oxalis?

Either by putting it in the SBDH (yet CEF informs me this should not be added, the Connectivity test showed me differently),
or by configuring it in any other way.

**Comment by Kim-Billit** [comment]
Anyone?

**Comment by amaters-easy** [comment]
we were able to get almost all this info in the soap message using:

`<BusinessScope>
			<Scope>
				<Type>DOCUMENTID</Type>
				<Identifier/>
<InstanceIdentifier>connectivity::cef##connectivity::TC1Leg1</InstanceIdentifier>
			</Scope>
			<Scope>
				<Type>PROCESSID</Type>
				<Identifier>e-delivery</Identifier>
				<InstanceIdentifier>bdx:noprocess</InstanceIdentifier>
			</Scope>
		</BusinessScope>
`
Except for the <ns:To><ns:PartyId> which seems to get always be filled by the used certificate
